src/var/binary_snps.jl

```diff
@@ -0,0 +1,167 @@
+
+#=
+Objects of type BinarySNPs store the SNPs of an individual
+The name BinarySNPs is double edged: it refers to both the fact that
+SNPs are stored in a binary format, and also that the SNPs are binary,
+in that they have two states, a reference state and an alternate state.
```

Comment:
I had considered going a slightly alternate route and rather than having multiple binary vectors for diploids and so on. Going the BioSequence route and making some alphabet, where the number of bits used to store each snp is the ploidy of the individual.